src/koppeling-dialog/koppeling-dialog.html

```diff
@@ -0,0 +1,42 @@
+<template>
+  <require from="./koppeling-dialog.css"></require>
+  <require from="../expander/expander"></require>
+  <require from="../expander/expander-container"></require>
+
+  <ux-dialog class="oe-dialog koppeling-dialog">
+    <ux-dialog-header show-close-button.bind="true">
+      <h3>Koppelingen</h3>
+    </ux-dialog-header>
+
+    <ux-dialog-body>
+      <div if.bind="loading" class="text-center">
+        <span class="spinner fa fa-spinner fa-pulse"></span> Er wordt gezocht naar referenties.
+      </div>
+
+      <div if.bind="!loading">
+        <p class="total-ref-tekst">${totalReftekst}</p>
+        <template if.bind="isExtern">
+          <i class="fa fa-info-circle"></i> Sommige referenties zijn voor jou niet zichtbaar op basis van je rechten.
+        </template>
+        <expander-container>
+          <template repeat.for="application of applications">
+            <expander header.bind="application.title + ' (' + application.count + ' referenties)'" expanded.bind="false" class="application-expander">
+              <template replace-part="expander-content">
+                <ul if.bind="application.count" class="oe-dialog-list">
+                  <li repeat.for="item of application.items" class="oe-dialog-list-item">
+                    <i class="fa fa-angle-right"></i>
+                    <a href="${item.uri}" target="_blank">${item.title}</a>
+                  </li>
+                </ul>
+              </template>
```

Comment:
kan application.count undefined zijn?
in dat geval: 
```js 
header.bind="application.title + ' (' + application.count || 0 + ' referenties)'"
```
en anders mss voor de duidelijkheid 
```js 
<ul if.bind="application.count > 0" 
```